src/core/game/Game.ts

```
@@ -184,6 +189,14 @@ export interface UnitParamsMap {
     lastSetSafeFromPirates?: number;
   };
 
+  [UnitType.TrainEngine]: {
+    targetUnit: Unit;
+  };
+
+  [UnitType.TrainCarriage]: {};
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)